shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -1479,6 +1480,36 @@ - (void)testMouseSupport API_AVAILABLE(ios(13.4)) {
       dispatchPointerDataPacket:std::make_unique<flutter::PointerDataPacket>(0)];
 }
 
+- (void)testPencilSupport API_AVAILABLE(ios(13.4)) {
+  if (@available(iOS 13.4, *)) {
+    // noop
+  } else {
+    return;
+  }
+
+  FlutterViewController* vc = [[FlutterViewController alloc] initWithEngine:self.mockEngine
+                                                                    nibName:nil
+                                                                     bundle:nil];
+  XCTAssertNotNil(vc);
+
+  id mockPencilInteraction = OCMClassMock([UIPencilInteraction class]);
+  XCTAssertNotNil(mockPencilInteraction);
+
+  [vc pencilInteractionDidTap:mockPencilInteraction];
+
+  flutter::PointerData pointer_data;
+
+  pointer_data.kind = flutter::PointerData::DeviceKind::kStylus;
+  pointer_data.signal_kind = flutter::PointerData::SignalKind::kStylusAuxiliaryAction;
+  pointer_data.preferred_auxiliary_stylus_action =
+      flutter::PointerData::PreferredStylusAuxiliaryAction::kIgnore;
```

Comment:
Where was `ignore` mocked?

Reply:
Ignore is the default value of UIPencilInteraction.preferredAction - it's a readonly property so I'm not sure how I would mock it to be a different enum.

Reply:
You can look into how to mock static methods using ocmock.

Reply:
> Ignore is the default value of UIPencilInteraction.preferredAction

We wouldn't want this test to fail based on real settings on the target device.

Reply:
See https://ocmock.org/reference/#mocking-class-methods
It would be something like:
```objc
OCMStub([mockPencilInteraction preferredAction]).andReturn(UIPencilPreferredActionIgnore);
```
And at the end of the test, reset the class method mock:
```objc
[mockPencilInteraction stopMocking];
```